server/manager/src/main/java/org/apache/accumulo/manager/upgrade/SplitRecovery12to13.java

```diff
@@ -67,37 +58,6 @@ public class SplitRecovery12to13 {
 
   private static final Logger log = LoggerFactory.getLogger(SplitRecovery12to13.class);
 
-  public static void addNewTablet(ServerContext context, KeyExtent extent, String dirName,
```

Comment:
The two method that were removed from the class were moved to the integration test because it was the only thing using them.  Thought it made it more clear that this code was only used by test code.